seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/commit/HiveSinkAggregatedCommitter.java

```diff
@@ -51,11 +54,21 @@ public HiveSinkAggregatedCommitter(
         this.tableName = tableName;
         this.abortDropPartitionMetadata =
                 readonlyConfig.get(HiveSinkOptions.ABORT_DROP_PARTITION_METADATA);
+        this.overwrite = readonlyConfig.get(HiveSinkOptions.OVERWRITE);
     }
 
     @Override
     public List<FileAggregatedCommitInfo> commit(
             List<FileAggregatedCommitInfo> aggregatedCommitInfos) throws IOException {
+        log.info("Aggregated commit infos size: {}", aggregatedCommitInfos.size());
+        for (FileAggregatedCommitInfo info : aggregatedCommitInfos) {
+            log.info("Commit info: {}", info);
+        }
+        log.info("overwrite: {}", overwrite);
+        // delete if overwrite
+        if (overwrite) {
+            deleteDirectories(aggregatedCommitInfos);
+        }
```

Comment:
```suggestion
        log.info("Aggregated commit infos: {}", aggregatedCommitInfos);
        if (overwrite) {
            log.info("start delete directories in aggregatedCommitInfos because overwrite is enabled.");
            deleteDirectories(aggregatedCommitInfos);
        }
```

Reply:
Thank you for your suggestion. I will delete the redundant logs as you suggested.